packages/core/src/server/devServer.ts

```diff
@@ -180,6 +189,14 @@ export async function createDevServer<
       );
     }
 
+    compiler?.hooks.run.tap('rsbuild:run', () => {
```

Comment:
Do we need to listen `hooks.run` for dev server? I guess `hooks.run` will never be called